autopush_rs/src/util/mod.rs

```diff
@@ -92,6 +93,24 @@ pub fn init_logging(json: bool) {
     slog_stdlog::init().ok();
 }
 
+/// Get the time since the UNIX epoch in seconds
+pub fn sec_since_epoch() -> u64 {
+    Utc::now().timestamp() as u64
+}
+
+/// Get the time since the UNIX epoch in seconds
+pub fn ms_since_epoch() -> u64 {
+    let now = Utc::now();
+    (now.timestamp() as u64) * 1_000 + (now.timestamp_millis() as u64)
```

Comment:
you meant timestamp_subsec_millis here, similar to us_since_epoch's impl. However timestamp_millis does all of this for us, so this whole fn should simply be: `Utc::now().timestamp_millis() as u64`